### PR TITLE
Update copyright for 2018

### PR DESCRIPTION
**Pull request by abrauchli**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,8 @@ sw_objects := ${sensirion_common_dir}/sw_i2c/sensirion_sw_i2c.o ${sensirion_comm
 hw_objects := ${sensirion_common_dir}/hw_i2c/sensirion_hw_i2c_implementation.o
 all_objects := ${common_objects} ${hw_objects} ${sw_objects} shtc1.o sht3x.o
 
-binaries := example_usage_shtc1_sw example_usage_shtc1_hw example_usage_sht3x_sw example_usage_sht3x_hw
+binaries := example_usage_shtc1_sw example_usage_shtc1_hw \
+            example_usage_sht3x_sw example_usage_sht3x_hw
```
Comment (rnestler) [inline Makefile]: Doesn't quite fit the commit message

File: Makefile [hunk 2]
```diff
@@ -45,4 +46,4 @@ release:
 	done
 
 clean:
-	rm -rf ${all_objects} ${binaries} release
+	rm -rf ${all_objects} ${binaries} release.pyc release
```
Comment (rnestler) [inline Makefile]: Same here.

Comment (abrauchli) [reply Makefile]: Intentional.. I'll update the message

File: Makefile (hunk 1 above)
Comment (abrauchli) [reply Makefile]: Attentive reviewer :D Intentional.. I'll update the message

**Pull request merged**